kratos/tests/cpp_tests/sources/test_model_part.cpp

```diff
@@ -133,6 +133,15 @@ namespace Kratos {
         KRATOS_CHECK_IS_FALSE(r_model_part.HasNodalSolutionStepVariable(PRESSURE));
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(ModelPartFullName, KratosCoreFastSuite)
+    {
+        Model current_model;
+        ModelPart& r_model_part = current_model.CreateModelPart("Main");
+        ModelPart& r_sub_model_part = r_model_part.CreateSubModelPart("SubModelPart");
```

Comment:
Would you mind to test an extra level (subsubmodelpart) to ensure that the recursivity always work?

Reply:
Done